resources/js/components/shared/TimelinePlot.vue

```diff
@@ -241,12 +260,20 @@ export default {
           let [x_val, y_val] = Object.values(d).map(String);
           // render the html div element with this point's data
           let content =
-            `<b>${line_names[k]}</b>: ${y_val}
-            <br/>
-            <b>${labels.x_axis}</b>: ${x_val}`;
+            `<span class=tooltip-row1>
+               <b></b>: <text></text>
+             </span>
+             <br/>
+             <span class=tooltip-row2>
+               <b></b>: <text></text>
+             </span>`;
           tooltip.html(content) // add slight offset for visiblity
             .style("left", (d3.event.pageX + 15)+"px")
             .style("top", (d3.event.pageY - 10)+"px");
+          tooltip.select(".tooltip-row1").select("b").text(line_names[k]);
+          tooltip.select(".tooltip-row1").select("text").text(y_val);
+          tooltip.select(".tooltip-row2").select("b").text(xLabel);
+          tooltip.select(".tooltip-row2").select("text").text(x_val);
```

Comment:
I created more Vue refs to handle this, which is fine because there are only ever these four fields that we want to update. However, the template now has a total of 5 refs, and I feel like any more than that will quickly become messy and annoying to maintain.